cmd/minishift/cmd/start.go

```diff
@@ -265,7 +265,8 @@ func runStart(cmd *cobra.Command, args []string) {
 		fmt.Println("-- OpenShift cluster will be configured with ...")
 		fmt.Println("   Version:", requestedOpenShiftVersion)
 
-		err := cmdUtil.PullOpenshiftImageAndCopyOcBinary(dockerCommander, requestedOpenShiftVersion)
+		valid, _ := openshiftVersion.IsGreaterOrEqualToBaseVersion(requestedOpenShiftVersion, constants.RefactoredOcVersion)
+		err := cmdUtil.PullOpenshiftImageAndCopyOcBinary(dockerCommander, requestedOpenShiftVersion, valid)
```

Comment:
@praveenkumar IMO we should move the logic of deciding `isGreaterOrEqualToBaseVersion` inside of `PullOpenshiftImageAndCopyOcBinary`

Reply:
Changing a function signature is completely OK since this is not something changes to UX side. I don't have any strong option about this to be part of `PullOpenshiftImageAndCopyOcBinary` method or here.

Reply:
it would make it consolidated